docs/custodian.md

```diff
@@ -0,0 +1,279 @@
+I0319 11:19:05.443371 1243525 sdp.go:133] Sending jump address to 0x8000f400
+I0319 11:19:05.443835 1243525 sdp.go:138] Serial download on /dev/hidraw0 complete
+I0319 11:19:06.444384 1243525 recovery.go:50] Witness device booting recovery image
+I0319 11:19:06.444418 1243525 recovery.go:105] Waiting for block device to appear
+I0319 11:19:09.884108 1243525 recovery.go:118] Waiting for block device to settle...
+I0319 11:19:14.900987 1243525 main.go:233] ✅ Detected device "/dev/hidraw0"
+I0319 11:19:14.901050 1243525 main.go:234] ✅ Detected blockdevice /dev/disk/by-id/usb-F-Secure_USB_armory_Mk_II_CA6B65D9D4992516-0:0
+I0319 11:19:14.905938 1243525 main.go:373] Found config at block 0x4fb0
+I0319 11:19:14.905964 1243525 main.go:378] Reading 0x2d6c00 bytes of firmware from MMC byte offset 0x400
+I0319 11:19:15.054263 1243525 main.go:373] Found config at block 0x5000
+I0319 11:19:15.054359 1243525 main.go:378] Reading 0xdd44e8 bytes of firmware from MMC byte offset 0x20505000
+I0319 11:19:15.749003 1243525 main.go:373] Found config at block 0x200000
+I0319 11:19:15.749041 1243525 main.go:378] Reading 0x102d8c0 bytes of firmware from MMC byte offset 0x4000a000
+I0319 11:19:16.594640 1243525 main.go:294]   ✅ Bootloader: proof bundle is self-consistent
+I0319 11:19:16.603331 1243525 main.go:317]   ✅ Bootloader: proof bundle checkpoint(@10) is consistent with current view of log(@17)
+I0319 11:19:16.636720 1243525 main.go:294]   ✅ TrustedOS: proof bundle is self-consistent
+I0319 11:19:16.636804 1243525 main.go:317]   ✅ TrustedOS: proof bundle checkpoint(@17) is consistent with current view of log(@17)
+I0319 11:19:16.658899 1243525 main.go:294]   ✅ TrustedApplet: proof bundle is self-consistent
+I0319 11:19:16.659019 1243525 main.go:317]   ✅ TrustedApplet: proof bundle checkpoint(@12) is consistent with current view of log(@17)
+I0319 11:19:16.659029 1243525 main.go:128] ✅ Device verified OK!
+I0319 11:19:16.659034 1243525 main.go:129] ----------------------------------------------------------------------------------------------
+I0319 11:19:16.659039 1243525 main.go:130] 🔷🔷🔷 🙋 OPERATOR: please ensure boot switch is set to MMC, and then reboot device 🙏
+I0319 11:19:16.659054 1243525 main.go:131] ----------------------------------------------------------------------------------------------
+```
+
+The block of ✅ green ticks towards the end indicates that the firmware on the device was successfully verified.
+
+If you see these green ticks when running against your device, then you can be sure that:
+
+1. Your ArmoredWitness has not been tampered with on its journey from us to you.
+1. All 3 pieces of firmware on the device are publicly discoverable via the transparency log.
+   * This enables anyone running the [verify_build](/cmd/verify_build) tool to check that all the firmware in the
+     log is reproducibly built, and, consequently, to inspect the source code for all released versions of the firmware to see that it's doing only what it claims.
```

Comment:
Would it be worth adding a link to the Claimant Model here?

Reply:
I was consciously shooting for a middle ground of "here are some of the details, but not so much as to overwhelm", but a "more info..." section of links at the bottom would be a good place to link to that from, wdyt?